### ISSUE TITLE
CI sometimes hanging on tools/format.ts

### ISSUE THREAD

**Issue by ry** [denoland/deno#1718]
This doesn't seem to happen with any regularity - not sure what's happening.

Example: https://ci.appveyor.com/project/deno/deno/builds/22231080
```
python tools\lint.py
Done processing libdeno\modules_test.cc
Done processing libdeno\binding.cc
Done processing libdeno\api.cc
Done processing libdeno\exceptions.cc
Done processing libdeno\test.h
Done processing libdeno\test.cc
Done processing libdeno\deno.h
Done processing libdeno\file_util_test.cc
Done processing libdeno\exceptions.h
Done processing libdeno\internal.h
Done processing libdeno\file_util.h
Done processing libdeno\file_util.cc
Done processing libdeno\modules.cc
Done processing libdeno\snapshot_creator.cc
Done processing libdeno\libdeno_test.cc
python C:\deno\third_party\cpplint\cpplint.py --filter=-build/include_subdir --repository=libdeno --extensions=cc,h --recursive libdeno
node C:\deno\third_party\node_modules\tslint\bin\tslint -p . --exclude **/gen/**/*.ts
node C:\deno\third_party\node_modules\tslint\bin\tslint ./js/**/*_test.ts ./tests/**/*.ts --exclude **/gen/**/*.ts --project tsconfig.json
C:\Python27-x64\python.exe third_party/depot_tools/pylint.py tools\benchmark.py tools\benchmark_test.py tools\build.py tools\build_test.py tools\deno_dir_test.py tools\docs.py tools\fmt_test.py tools\http_benchmark.py tools\http_server.py tools\integration_tests.py tools\is_tty_test.py tools\lint.py tools\permission_prompt_test.py tools\prebuilt.py tools\prefetch_test.py tools\repl_test.py tools\run_node.py tools\setup.py tools\setup_test.py tools\sha256sum.py tools\sync_third_party.py tools\test.py tools\test_format.py tools\third_party.py tools\throughput_benchmark.py tools\unit_tests.py tools\upload_website.py tools\util.py tools\util_test.py build_extra\rust\get_cargo_info.py build_extra\rust\get_rustc_info.py build_extra\rust\run.py
python tools\test_format.py
C:\deno\target\release\deno.exe --allow-run tools/format.ts
Command exited with code -1073741819
```

**Comment by ry** [comment]
theory: it's running the debug build on a very large JS files and hanging because of all the extra checks V8 is doing
```
> du -sh js/deps/https/deno.land/x/std/prettier/vendor/
3.3M	js/deps/https/deno.land/x/std/prettier/vendor/
```

**Comment by kitsonk** [comment]
Closed by #1752 (for now)?
